opencog/atoms/base/Atom.h

```diff
@@ -248,6 +248,17 @@ class Atom
         setAttentionValue(new_av);
     }
 
+    /** Change the Very-Long-Term Importance */
+    void setVLTI(AttentionValue::vlti_t vltiValue)
+    {
```

Comment:
VLTI was designed to not be settable, only incrementable/decrenmentable.  I don't recall why.